uber/models.py

```diff
@@ -379,6 +379,82 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
             if not ignore_csrf:
                 check_csrf(params.get('csrf_token'))
 
+    def timespan(self, minute_increment=60):
```

Comment:
just to clarify: what I (confusingly) said here was we should probably go for something of the form ```{{ checkgroup(attendee, 'requested_depts') }}```